### DISCUSSION TITLE
Typeform for GitHub Student Developers

### DISCUSSION THREAD

**Discussion by SirajiMoto** [community/community#62589] [GitHub Education]
### Hi everyone,

Cant claim the benefit for Typeform just get directed to this link

[](https://tf-gh-edu.glitch.me/auth/typeform/callback?code=6BHqfj18eKQT37hu2bJjvK6pQQZr7s2f8iHo9YrQCGeE&state=vZzAehqGYwp9zoGf8V8c5wCD)

here is the message

> Site didn't respond
> Something in the code caused it to close the connection before providing a response.
> 
> If this is your project please visit us at [support.glitch.com](https://support.glitch.com/) for assistance.

i have check glitch support  they say i should ask here to Github developer pack support

**Comment by Rhodless** [comment]
I am having the same issue

**Comment by sergmetelin** [comment]
Hi there, bumping this up. Typeform offer just plainly doesn't work. Please either fix or remove it! Thanks!
